subsys/net/ip/net_shell.c

```diff
@@ -783,6 +783,31 @@ static void print_tc_rx_stats(const struct shell *shell, struct net_if *iface)
 	int i;
 
 	PR("RX traffic class statistics:\n");
+
+#if defined(CONFIG_NET_PKT_RXTIME_STATS)
+	PR("TC  Priority\tRecv pkts\tbytes\ttime\n");
```

Comment:
Would it make sense to specify time unit straight in the header here?

Reply:
On closer look, I see that each table row has "us" for that column (code below). Not sure which way is better, but that addresses this concern.